src/computed/wallet.js

```diff
@@ -21,13 +21,13 @@ const ComputedWallet = store => {
         balanceSatoshis,
         pendingBalanceSatoshis,
         channelBalanceSatoshis,
-        limboBalanceSatoshis,
+        channelBalanceClosingSatoshis,
```

Comment:
Cool, this does make sense to fix the [double-add problem](https://github.com/lightninglabs/lightning-app/issues/1016#issuecomment-472725500). Between this and `pendingBalanceSatoshis` we cover all the pending funds, then `balanceSatoshis` is on-chain proper, and `channelBalanceSatoshis` is off-chain proper.